src/parser/mod.rs

```diff
@@ -15188,6 +15137,11 @@ impl<'a> Parser<'a> {
                 let func_name = self.parse_object_name(true)?;
                 let func = self.parse_function(func_name)?;
                 return Ok(Some(TableVersion::Function(func)));
+            } else if dialect_of!(self is DatabricksDialect)
```

Comment:
I think this should use the defined `supports_timestamp_versioning`?

Reply:
Ah! That makes sense, thank you for explaining. I believe this is addressed by the following 2 commits:
* https://github.com/apache/datafusion-sqlparser-rs/pull/2134/commits/21783cc67360cc4e0a53f91cd04349b668fcbc33
* https://github.com/apache/datafusion-sqlparser-rs/pull/2134/commits/270c41b9ac6c46b396dc47ec435f1a37afe6e883